src/autotype/AutoType.cpp

```diff
@@ -396,6 +396,9 @@ QList<AutoTypeAction*> AutoType::createActionFromTemplate(const QString& tmpl, c
     else if (tmplName.compare("enter",Qt::CaseInsensitive)==0) {
         list.append(new AutoTypeKey(Qt::Key_Enter));
     }
+    else if (tmplName.compare("space",Qt::CaseInsensitive)==0) {
+        list.append(new AutoTypeChar(' '));
```

Comment:
I've tested the `Qt::Key_Space` but returned the error `No such key: keysym=0x0` (on GNU/Linux)
So I opted for `Char(' ')` that works fine.